sdk/typespec/typespec_client_core/src/http/response.rs

```diff
@@ -279,6 +327,12 @@ impl ResponseBody {
     }
 }
 
+impl Model for ResponseBody {
```

Comment:
We could. We also don't _need_ to do it, but it means a plain `Response<ResponseBody>` would have no `into_body` method. Instead, it would have only `into_{raw,json,xml}_body` methods.

How about I try removing it for now, we can restore it later if it's proving inconvenient. It's non-breaking to add later, but breaking to remove later.

Reply:
We also get it from `into_raw_body()`, right? In which case I think it's explicit. Yeah, I like the idea of not implementing `Model` for now but we could keep it in our back pocket.